The first symbol no longer exists; the second is now private.

XXX file upstream bug
--- totem-pl-parser-3.10.5/plparse/plparser.symbols	2011-09-18 03:33:29.000000000 -0700
+++ totem-pl-parser-3.10.5/plparse/plparser.symbols	2015-07-05 19:59:07.454489580 -0700
@@ -4,7 +4,6 @@
 totem_cd_get_human_readable_name
 totem_cd_has_medium
 totem_cd_mrl_from_type
-totem_disc_media_type_get_type
 totem_disc_media_type_quark
 totem_pl_parser_add_ignored_mimetype
 totem_pl_parser_add_ignored_scheme
@@ -14,7 +13,6 @@
 totem_pl_parser_error_get_type
 totem_pl_parser_error_quark
 totem_pl_parser_get_type
-totemplparser_marshal_VOID__STRING_STRING_STRING
 totem_pl_parser_new
 totem_pl_parser_parse
 totem_pl_parser_parse_async
